src/main/kotlin/no/nav/familie/ba/sak/kjerne/steg/grunnlagForNyBehandling/VilkårsvurderingForNyBehandlingService.kt

```diff
@@ -179,6 +184,12 @@ class VilkårsvurderingForNyBehandlingService(
                         ?.map { it.aktør } ?: emptyList(),
             )
 
+        if (unleashService.isEnabled(FeatureToggle.PREUTFYLLING_VILKÅR.navn, false)) {
```

Comment:
```suggestion
        if (unleashService.isEnabled(FeatureToggle.PREUTFYLLING_VILKÅR)) {
```
Bruk enum, ikke `.navn`. Default `defaultValue` er `false`.